addon/chart-data-updater.js

```diff
@@ -36,8 +36,14 @@ export default Ember.Object.extend({
           item = item || 0;
           if(typeof chartDataset.bars !== 'undefined') {
             chartDataset.bars[j].value = item;
+
+            // Update tooltip labels
+            chartDataset.bars[j].label = labels[j];
           } else {
             chartDataset.points[j].value = item;
+
+            // Update tooltip labels
+            chartDataset.points[j].label = labels[j];
```

Comment:
Rather than call this twice (in both paths of the if condition), why not just call it once after the if condition has resolved?

```
if(typeof chartDataset.bars !== 'undefined') {
  chartDataset.bars[j].value = item;
} else {
  chartDataset.points[j].value = item;
}
// Update tooltip labels
chartDataset.points[j].label = labels[j];
```
Since it seems we always want to make this call.